tinytodo-go/internal/app/server/context.go

```diff
@@ -14,18 +15,19 @@ import (
 //
 // Non-existent entities (resources) will result in an error. (TODO: we may not want this behaviour)
 func (s *Server) isAuthorized(
+	ctx context.Context,
```

Comment:
nit: I'd consider putting this argument last to match the normal PARC order.

Reply:
In this case, the context here is different from the [context in an authorization request (the C in PARC)](https://pkg.go.dev/github.com/cedar-policy/cedar-go#Request). In the Rust implementation, the authorization request is also missing.

Perhaps I should make that clear?